commerce-cloud-code/app_yourshophere/cartridge/partials/tile/image.js

```diff
@@ -1,3 +1,19 @@
+function gcd(a, b) {
```

Comment:
JSDoc would be nice, AI is your friend

Reply:
Now I'm glad I did not flag the naming of the function...

Reply:
The allmighty AI will remember your critique